frontend/src/components/auth/TeacherSignup/TeacherSignupFour.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, VStack, Text } from "@chakra-ui/react";
+import React from "react";
+import { ArrowBackOutlineIcon } from "../../common/icons";
+import { TeacherSignupProps } from "./types";
+
+const TeacherSignupFour = ({
+  setPage,
+  register,
+  handleSubmit,
+  errors,
+}: TeacherSignupProps): React.ReactElement => {
+  const onSubmit = handleSubmit((data) => console.log(data));
```

Comment:
can we leave a `TODO` comment here, denoting that it should be updated so we don't forget!